src/components/DateInput.jsx

```diff
@@ -232,7 +234,10 @@ class DateInput extends React.PureComponent {
           value={value}
           onChange={this.onChange}
           onKeyDown={this.onKeyDown}
-          onFocus={onFocus}
+          onFocus={e => {
+            setInputHeight(e.target.getBoundingClientRect().y - FANG_HEIGHT_PX - 3);
+            onFocus(e);
+          }}
```

Comment:
Looks like they have methods on the class. To be consistent, can we have a `this.onFocus`?